pkg/network/ebpf/c/tracer/maps.h

```diff
@@ -60,7 +60,7 @@ BPF_HASH_MAP(udp_sendpage_args, __u64, struct sock *, 1024)
  * Map to hold struct sock parameter for tcp_recvmsg/tcp_read_sock calls
  * to be used in kretprobe/tcp_recvmsg/tcp_read_sock
  */
-BPF_HASH_MAP(tcp_recvmsg_args, __u64, struct sock *, 0)
+BPF_HASH_MAP(tcp_recvmsg_args, __u64, struct sock *, 2048)
```

Comment:
Isn't this a constant in userspace? We should set it there instead imo.

Reply:
I was under the impression our paradigm was to set constants here, and then anything dynamic we would set in userspace (the other changed file in this PR)

Reply:
that makes sense. I can rewrite this to set the size in userspace as a function of maxactive